docs/guides/Integration-Guides/headless-commerce/headless-cart-and-checkout.md

```diff
@@ -33,155 +36,431 @@ Items are the products of a shopping cart. In the `orderForm`, items' informatio
+
+The request body used in this step is based on the order's paymentData attachment. You can get this data from the response of the [Get client profile by email](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pub/profiles?endpoint=get-/api/checkout/pub/profiles), as in the example.
+
+```json Request
+"paymentData":{
+   "updateStatus":"updated",
+   "installmentOptions":[
+      {
+         "paymentSystem":2,
+         "bin":null"paymentName":null"paymentGroupName":null"value":65780,
+         "installments":[
+            {
+               "count":1,
+               "hasInterestRate":false,
+               "interestRate":0,
+               "value":65780,
+               "total":65780,
+               "sellerMerchantInstallments":[
+                  {
+                     "id":"MYSHOP",
+                     "count":1,
+                     "hasInterestRate":false,
+                     "interestRate":0,
+                     "value":65780,
+                     "total":65780
+                  }
+               ]
+            }
+         ]
+      }
+   ]
+}
+```
+
+Learn more about these endpoints:
 
-- [Send payment information API request (public)](https://developers.vtex.com/docs/api-reference/payments-gateway-api#post-/api/pub/transactions/-transactionId-/payments)
-- [Send payment information API request (private)](https://developers.vtex.com/docs/api-reference/payments-gateway-api#post-/api/pvt/transactions/-transactionId-/payments).
+- [`POST` Send payment information](https://developers.vtex.com/docs/api-reference/payments-gateway-api#post-/api/pub/transactions/-transactionId-/payments)
+- [`POST` Send payment information with a saved credit card](https://developers.vtex.com/docs/api-reference/payments-gateway-api#post-/api/pvt/transactions/-transactionId-/payments)
 
-The body of the request used in this step is based in the order's `paymentData` attachment. You can get this data from the response of the [order placement](#placing-orders) request.
+> ❗ Make sure that all value-related fields match the information sent when [placing the order](#place-order) to create a successful request.
 
-Learn more about these API requests:
+---
 
-- [Send payment information](https://developers.vtex.com/docs/api-reference/payments-gateway-api#post-/api/pub/transactions/-transactionId-/payments)
-- [Send payment information with a saved credit card](https://developers.vtex.com/docs/api-reference/payments-gateway-api#post-/api/pvt/transactions/-transactionId-/payments)
+#### Retrieving saved credit cards
+
+In the payment section at checkout, we recommend retrieving the saved credit card information from the customer profile. This can ease the checkout process as the customer would only need to select the chosen credit card and input the CVV (Card Verification Value). This process will only display the last four numbers of a credit card to the customer, making it a secure interaction with the shopper data.
```

Comment:
```suggestion
In the payment section at checkout, we recommend retrieving the saved credit card information from the customer profile. This can ease the checkout process as the customer would only need to select the chosen credit card and input the CVV (Card Verification Value). This process will only display the last four numbers of a credit card to the customer, making it a secure interaction with the client data.
```